CONTRIBUTING.md

```diff
@@ -121,7 +121,7 @@ Before attempting to make any changes, be sure to read the following Wiki pages:
 
 ### Check Before Doing Anything
 
-It's important that you look through any open issues or PRs in a repo before attempting to submit a new issue or work on a change, regardless of the complexity. This will help avoid any duplicates from being made, as well as prevent more than one person working on the same thing at the same time.
+It's important that you look through any open [issues](https://github.com/TheOdinProject/theodinproject/issues?page=1&q=is%3Aissue+is%3Aopen) or [PRs](https://github.com/TheOdinProject/theodinproject/pulls) in a repo before attempting to submit a new issue or work on a change, regardless of the complexity. This will help avoid any duplicates from being made, as well as prevent more than one person working on the same thing at the same time.
```

Comment:
```suggestion
It's important that you look through any open [issues](https://github.com/TheOdinProject/theodinproject/issues) or [pull requests](https://github.com/TheOdinProject/theodinproject/pulls) in a repo before attempting to submit a new issue or work on a change, regardless of the complexity. This will help avoid any duplicates from being made, as well as prevent more than one person working on the same thing at the same time.
```

Reply:
Two minor suggestions - we can make the issue link a little shorter / cleaner and have it still point to the correct spot, and in the spirit of being more explicit / beginner friendly it might be a good chance to expand the 'PR' acronym.

What do you think?

Reply:
I didn't realize that the issues link would default to only open issues. I agree that's better. And I guess it makes sense that it does that, it's behaving the same as the default PR link. (And I tested it in an incognito window to make sure that's really what it's defaulting to, not that it was saving some kind of previous state or user preference.)

I also agree that using "pull request" instead of "PR" is more explicit and beginner friendly.

Thanks for the suggestions.